### DISCUSSION TITLE
1.0.2

### DISCUSSION THREAD

**Discussion by Schwartzblat** [Schwartzblat/WhatsApp-Web-Plus#18] [Announcements]
- Fix view once messages by using hook on protobuf.

<hr /><em>This discussion was created from the release <a href='https://github.com/Schwartzblat/WhatsApp-Web-Plus/releases/tag/1.0.2'>1.0.2</a>.</em>

**Comment by Blonkyblonk** [comment]
Still can't view once message

**Reply by Schwartzblat** [reply]
Working on that.
